modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client.cli/src/main/scala/edu/gemini/seqexec/web/client/cli/SeqexecTerminal.scala

```diff
@@ -102,17 +103,17 @@ object SeqexecTerminal extends js.JSApp {
   }
 
   object ShowHandler extends CommandHandler {
-    def width(ks: List[StepConfig]): Int = ks match {
-      case Nil => 0
-      case _   => ks.map(_.key.length).max
+    def width(ks: StepConfig): Int = ks match {
+      case m if m.isEmpty => 0
+      case _              => ks.map(_._2.length).max
     }
```

Comment:
I don't have any opinion about this, but is it preferable to an `if ... else ...` statement? I frequently find myself hesitating about what to pick.